package/kedro_viz/integrations/kedro/hooks.py

```diff
@@ -135,7 +135,9 @@ def get_file_size(self, dataset: Any) -> Union[int, None]:
             return None
 
         try:
-            file_path = get_filepath_str(Path(dataset._filepath), dataset._protocol)
+            file_path = get_filepath_str(
+                PurePosixPath(dataset._filepath), dataset._protocol
```

Comment:
I am confused, `Path` usually is the platform-agnostic way and works on both Windows and Linux. But PurePosixPath wouldn't work on Windows, so why are we changing this. Happy to get on a call to understand better.